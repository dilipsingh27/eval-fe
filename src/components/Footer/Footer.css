.footer {
    background-color: black;
    padding: 0 10%;
    height:100px;
    position:fixed;
    bottom: 0;
    width:100%;
}
.footer p {
    color: white;
    font-size: 20px;
    margin-top: 20px;
    margin-bottom: 20px;
}