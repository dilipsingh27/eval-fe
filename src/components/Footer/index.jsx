import React from 'react';
import './Footer.css';

const Footer = () => {
    return (
        <div className='footer'>
            <p className='footer-content'>THEMES</p>
        </div>
    );
};

export default Footer;