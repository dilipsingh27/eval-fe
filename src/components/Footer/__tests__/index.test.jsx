import { render,screen } from '@testing-library/react';
import Footer from '..'
import { BrowserRouter } from 'react-router-dom';

describe('Footer', () => { 
    it('should render without crashing', () => {
        render(<BrowserRouter><Footer/></BrowserRouter> );
        const appElement = screen.getByText('THEMES',{exact:false});
        expect(appElement).toBeInTheDocument();
    });
});

describe('Footer Snapshot', () => {
    it('should render forming a snapshot', () => {
        const { asFragment } = render(<BrowserRouter><Footer/></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot(); 
    });
});