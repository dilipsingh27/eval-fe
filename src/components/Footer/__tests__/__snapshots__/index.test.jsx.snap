// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot should render forming a snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p
      class="footer-content"
    >
      THEMES
    </p>
  </div>
</DocumentFragment>
`;
